scripts/run_backend_tests.py

```diff
@@ -13,27 +13,19 @@
 # limitations under the License.
 
 """Script for running backend tests in parallel.
-
```

Comment:
Why has this line been removed? Is this intentional?
Ditto for all the removed lines in this file.

Reply:
No this is not intentional at all. I had only made changes in en.json and qqq.json ( aligned some changes alphabetically) but somehow when I committed there were changes in more files as well. I do not know why this is happening. Can you help ?
I will revert these changes and update soon.